packages/vscode-common/getExtensionApi.ts

```diff
@@ -1,41 +1,42 @@
 import type {
   CommandServerApi,
+  ExcludableSnapshotField,
+  ExtraContext,
+  ExtraSnapshotField,
+  HatTokenMap,
   IDE,
   NormalizedIDE,
   SerializedMarks,
+  SnippetMap,
   TargetPlainObject,
+  TestCaseSnapshot,
   TextEditor,
 } from "@cursorless/common";
-import type {
-  ExcludableSnapshotField,
-  ExtraContext,
-  ExtraSnapshotField,
-  Graph,
-  TestCaseSnapshot,
-  ThatMark,
-} from "@cursorless/cursorless-engine";
 import * as vscode from "vscode";
 import type { SyntaxNode } from "web-tree-sitter";
-import type { SnippetMap } from "../cursorless-engine/snippets/snippet.types";
-import type { Target } from "../cursorless-engine/typings/target.types";
 
-interface TestHelpers {
-  graph: Graph;
+export interface TestHelpers {
   ide: NormalizedIDE;
   injectIde: (ide: IDE) => void;
 
-  // FIXME: Remove this once we have a better way to get this function
-  // accessible from our tests
-  plainObjectToTarget(
+  hatTokenMap: HatTokenMap;
+
+  commandServerApi: CommandServerApi;
+
+  toVscodeEditor(editor: TextEditor): vscode.TextEditor;
+
+  setThatMark(
     editor: vscode.TextEditor,
-    plainObject: TargetPlainObject,
-  ): Target;
+    targets: TargetPlainObject[] | undefined,
+  ): void;
+  setSourceMark(
+    editor: vscode.TextEditor,
+    targets: TargetPlainObject[] | undefined,
+  ): void;
 
   // FIXME: Remove this once we have a better way to get this function
   // accessible from our tests
   takeSnapshot(
-    thatMark: ThatMark | undefined,
-    sourceMark: ThatMark | undefined,
```

Comment:
See how we've removed these from signature of `takeSnapshot` exposed to test cases; we provide them ourselves here https://github.com/cursorless-dev/cursorless/pull/1280/files#diff-08aaf0534aa1822e2fe091bc20a807b628da66d0dfa64c85629f43fc7f5e2ffbR52-R53